crates/pglt_lexer/src/lib.rs

```diff
@@ -61,8 +61,18 @@ pub static WHITESPACE_TOKENS: &[SyntaxKind] = &[
     SyntaxKind::SqlComment,
 ];
 
-static PATTERN_LEXER: LazyLock<Regex> =
-    LazyLock::new(|| Regex::new(r"(?P<whitespace> +)|(?P<newline>\r?\n+)|(?P<tab>\t+)").unwrap());
+static PATTERN_LEXER: LazyLock<Regex> = LazyLock::new(|| {
+    #[cfg(windows)]
+    {
+        // On Windows, treat \r\n as a single newline token
+        Regex::new(r"(?P<whitespace> +)|(?P<newline>\r\n|\n+)|(?P<tab>\t+)").unwrap()
```

Comment:
Need to check if this also works with multiple /r/n

Reply:
machste das noch in dem PR?